package.json

```diff
@@ -1,31 +1,15 @@
 {
 	"name": "soundcraft-ui",
-	"version": "2.0.6",
-	"api_version": "1.0.0",
-	"support": {
-		"action": true,
-		"feedback": true,
-		"preset": false
-	},
```

Comment:
I suppose this was okay to remove? It was not part of the list in the guide.

Reply:
I havent seen this in a module before

Reply:
Okay this is funny 😅 no idea where this came from